src/Compilers/CSharp/Portable/Binder/Semantics/OverloadResolution/OverloadResolution.cs

```diff
@@ -219,6 +220,102 @@ internal void MethodOrPropertyOverloadResolution<TMember>(
             }
         }
 
+#nullable enable
+        /// <returns>false if there are ambiguous candidates in the set</returns>
+        internal bool FilterMethodsForUniqueSignature(ArrayBuilder<MethodSymbol> methods, out bool useParams)
+        {
+            useParams = false;
+
+            if (methods.Count == 0)
+            {
+                return true;
+            }
+
+            var result = OverloadResolutionResult<MethodSymbol>.GetInstance();
+            var results = result.ResultsBuilder;
+            var useSiteInfo = CompoundUseSiteInfo<AssemblySymbol>.Discarded;
+
+            // Type arguments are verified in the caller.
+            var typeArguments = ArrayBuilder<TypeWithAnnotations>.GetInstance(0);
+
+            // We do not have any arguments when determining unique signature.
+            var arguments = AnalyzedArguments.GetInstance();
+
+            // Avoid passing reduced methods to overload resolution.
```

Comment:
>// Avoid passing reduced methods to overload resolution. [](http://example.com/codeflow?start=12&length=56)

The motivation for this change is not obvious especially that there are no new/modified tests.

Reply:
Is this in response to [https://github.com/dotnet/roslyn/pull/71380#pullrequestreview-2145410303](https://github.com/dotnet/roslyn/pull/71380#pullrequestreview-2145410303)?

Reply:
> Is this in response to [#71380 (review)](https://github.com/dotnet/roslyn/pull/71380#pullrequestreview-2145410303)?

Yes.